app/controllers/posts_controller.rb

```diff
@@ -26,6 +26,14 @@ def create
     redirect_to user_path(@current_user.id)
   end
 
```

Comment:
- [ ]  In order to actually be able to use cancancan, you need to initialize it on a controller.  In this case, you need to initialize it in the posts controller, at the beggining:

```ruby
  class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```